src/create-game.js

```diff
@@ -0,0 +1,11 @@
+// CREATE A NEW GAME
+fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: JSON.stringify({
+    name: 'Squid Game',
+  }),
+}).then((res) => res.json());
+// .then((data) => console.log(data.result));
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you remove this commented code. It can make your code hard to read.